packages/dataviews/src/components/dataviews-view-config/style.scss

```diff
@@ -0,0 +1,39 @@
+.dataviews-view-config {
+    width: 363px;
+    container-type: inline-size;
+    padding: $grid-unit-20;
+}
+
+.dataviews-settings-section__title {
+    font-size: 13px;
+	font-weight: 600;
+	margin-top: 0;
+    margin-bottom: $grid-unit-05;
+}
+
+.dataviews-settings-section.dataviews-settings-section {
+    align-items: start;
+    grid-template-columns: 1fr;
+}
+
+.dataviews-settings-section__description {
+	margin: -$grid-unit-10 0 $grid-unit-10 0;
+	font-size: $helptext-font-size;
+	font-style: normal;
+	color: $gray-700;
+}
```

Comment:
I'm a bit surprised by the number of styles that we need to make this look correct. Do we need all these "widths", "margins", "paddings"... Why our default components don't work well? What's missing as a UI component to render that dropdown?

Reply:
Reading the code again, it seems mostly about the `SettingsSection` component. I would suggest extract this component to its own folder `dataviews/src/components/fieldset` or something to start with and IMO, this should be a UI component like a `Fieldset` component or something. I also wonder if it should be using a `fieldset + legend` element (not entirely sure)

Reply:
We could be able to save some of these styles by using the `Text` component.